inlong-agent/bin/agent-env.sh

```diff
@@ -36,7 +36,7 @@ else
   export JPS="$JAVA_HOME/bin/jps"
 fi
 
-HEAP_OPTS="-Xmx1024m -Xms512m"
+HEAP_OPTS="-Xmx7168m -Xms7168m"
```

Comment:
this size is too large for docker, we should make the default value more appropriately.

Reply:
done